### ISSUE TITLE
Regarding model training

### ISSUE THREAD

**Issue by jiangchaokang** [ispc-lab/NeuralPCI#4]
I have observed that the model training process is all trained in a single batch. Why not use multiple batch training? Is there any significance to this? Looking forward to your answer.

**Comment by dyfcalid** [comment]
NeuralPCI is a per-scene fitting method due to the coordinate-based MLP, which is randomly initialized and then optimized for every single sample/sequence. This paradigm is similar to NeRF.

**Comment by jiangchaokang** [comment]
Thank you very much for your reply!
I have another small question:
Based on the above answer, does it mean that NeuralPCI did not use the training set of NL_Drive, but directly optimized for each sample on the NL_Drive test set before evaluation? Instead of first training the model on the NL_Drive training set, then loading the model weights, and then evaluating on each sample, with the final weights used to assess the samples. Because I didn't find the code to load the weights in the repo.

**Comment by dyfcalid** [comment]
Yes, there is no "training" process for NeuralPCI. It's an optimization-based method and directly applied to the test set.

**Comment by jiangchaokang** [comment]
Thank you for your time!
